.travis.yml

```diff
@@ -20,19 +20,27 @@ matrix:
         apt:
           packages:
           - ninja-build
-          - cmake
-      before install:
-        - sudo apt-get install libgtest-dev
-        - sudo apt-get install google-mock
+          - cmake  
+      # before install:
+        # - sudo apt-get install libgtest-dev
+        # - sudo apt-get install google-mock
       install:
-        - cd /usr/src/gtest
-        - sudo cmake CMakeLists.txt
-        - sudo make
-        - sudo cp *.a /usr/lib
-        - cd /usr/src/gmock
-        - sudo cmake CMakeLists.txt
+        # - cd /usr/src/gtest
+        # - sudo cmake CMakeLists.txt
+        # - sudo make
+        # - sudo cp *.a /usr/lib
+        # - cd /usr/src/gmock
+        # - sudo cmake CMakeLists.txt
```

Comment:
No need to clutter up with the commented lines. We have the commit history if we ever need to revert.